docs/contracts/index.md

```diff
@@ -486,3 +505,10 @@ The above example outputs the following
 >>> Properly raised AssertionError
 >>> Did not raise AssertionError
 ```
+
+### Contracts CLI Limitations
```

Comment:
Don't make this a separate section. Instead, you can make this a "note" in the CLI section.